tasks/meta.py

```diff
@@ -21,11 +22,88 @@
 from sqlalchemy.types import UserDefinedType
 
 from lib.logger import get_logger
+from logging import DEBUG
 import asyncpg
 
 LOGGER = get_logger(__name__)
 
 
+class CurrentSession(object):
```

Comment:
For Python 3 the `object` inheritance is not needed. But it makes it compatible with new-style classes in Python 2. Do we still support Python 2 in `bigmetadata`?

Reply:
I think that we haven't explicitly dropped support for Python 2, but I see no point in supporting both. What do you think, @antoniocarlon ?

Reply:
We migrated to Python 3 here https://github.com/CartoDB/bigmetadata/pull/344 so I would :fire: anything Python 2 related